railties/lib/rails/generators/rails/authentication/templates/models/user.rb

```diff
@@ -2,5 +2,5 @@ class User < ApplicationRecord
   has_secure_password
   has_many :sessions, dependent: :destroy
 
-  normalizes :email_address, with: -> { |e| e.strip.downcase }
+  normalizes :email_address, with: -> (e) { e.strip.downcase }
```

Comment:
Can you omit the parentheses?
We usually don't use parentheses for a single argument.

```
% git grep -n ' with: -> '
actionpack/lib/action_controller/metal/rate_limiting.rb:40:      #         by: -> { request.domain }, with: -> { redirect_to busy_controller_url, alert: "Too many signups on domain!" }, only: :new
actionpack/lib/action_controller/metal/rate_limiting.rb:47:      def rate_limit(to:, within:, by: -> { request.remote_ip }, with: -> { head :too_many_requests }, store: cache_store, **options)
actionpack/test/controller/rate_limiting_test.rb:13:  rate_limit to: 2, within: 2.seconds, by: -> { params[:rate_limit_key] }, with: -> { head :forbidden }, only: :limited_with
activerecord/lib/active_record/normalization.rb:18:    #     normalizes :email, with: -> email { email.strip.downcase }
activerecord/lib/active_record/normalization.rb:70:      #     normalizes :email, with: -> email { email.strip.downcase }
activerecord/lib/active_record/normalization.rb:71:      #     normalizes :phone, with: -> phone { phone.delete("^0-9").delete_prefix("1") }
activerecord/lib/active_record/normalization.rb:101:      #     normalizes :email, with: -> email { email.strip.downcase }
activerecord/test/cases/normalized_attribute_test.rb:9:    normalizes :name, with: -> name { name.titlecase }
activerecord/test/cases/normalized_attribute_test.rb:10:    normalizes :manufactured_at, with: -> time { time.noon }
activerecord/test/cases/normalized_attribute_test.rb:70:      normalizes :name, with: -> name { name&.titlecase || "Untitled" }, apply_to_nil: true
activerecord/test/cases/normalized_attribute_test.rb:88:      normalizes :name, with: -> name { name.reverse }
activerecord/test/cases/normalized_attribute_test.rb:97:      normalizes :name, with: -> name { name.succ }
guides/source/7_1_release_notes.md:91:  normalizes :email, with: -> email { email.strip.downcase }
guides/source/7_1_release_notes.md:92:  normalizes :phone, with: -> phone { phone.delete("^0-9").delete_prefix("1") }
railties/lib/rails/generators/rails/authentication/templates/controllers/sessions_controller.rb:3:  rate_limit to: 10, within: 3.minutes, only: :create, with: -> { redirect_to new_session_url, alert: "Try again later." }
railties/lib/rails/generators/rails/authentication/templates/models/user.rb:5:  normalizes :email_address, with: -> { |e| e.strip.downcase }
```

Reply:
👍